source/atlas-cli-changelog.txt

```diff
@@ -12,6 +12,31 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.1.7:
+
+{+atlas-cli+} 1.1.7
+-------------------
+
+Released 18 August 2022
+
+- Adds support for the following platforms:
+
+  - RHEL 9
+  - Ubuntu 22.04 (Jammy)
```

Comment:
I've never seen the version names mentioned before, so not sure about Jammy.

Reply:
We include the version names on the Apt tab of the [install page](https://www.mongodb.com/docs/atlas/cli/upcoming/install-atlas-cli/), but I can take it out, the version # will suffice. Thanks!